docs/pages/enterprise/sso.mdx

```diff
@@ -32,9 +36,149 @@ organization's Single Sign-On (SSO) provider.
   </Tile>
 </TileSet>
 
-## How does SSO work?
+## How Teleport uses SSO
+
+You can register your Teleport cluster as an application with your SSO provider.
+When a user signs in to Teleport, your SSO provider will execute its own
+authentication flow, then send an HTTP request to your Teleport cluster to
+indicate that authentication has completed.
+
+Teleport authenticates users to your infrastructure by issuing short-lived
+certificates. After a user completes an SSO authentication flow, Teleport issues
+a short-lived certificate to the user. Teleport also creates a temporary `user`
+resource on the Auth Service backend.
```

Comment:
I wonder if saying "user resource" spills implementation details a bit. How do you feel about just saying "user" everywhere? From the reader perspective it may be a bit more clear.

Reply:
That makes sense—I've removed "resource" unless we're specifically referring to the dynamic configuration resource (e.g., where we're indicating how to obtain a temporary user resource).